src/script/run-make.sh

```diff
@@ -111,6 +111,7 @@ EOM
     local c_compiler="${discovered_c_compiler}"
     local cmake_opts
     cmake_opts+=" -DCMAKE_CXX_COMPILER=$cxx_compiler -DCMAKE_C_COMPILER=$c_compiler"
+    cmake_opts+=" -DCMAKE_CXX_FLAGS=-std=gnu++20\ -isystem\ /usr/include/c++/11\ -isystem\ /usr/include/x86_64-linux-gnu/c++/11"
```

Comment:
I'm not sure I understand this, can you please explain the fix?
This seems to break `discover_compiler` logic. IIUC, this comes to fix a makecheck issue, discover_compiler should fallback to clang19 in our makecheck ci builders (ubuntu/deb). I don't think we should use gnu/gcc toolchain in this case.
From jenkins logs:
```
CMake Error: Unknown argument -isystem\
```

Reply:
No no it's not fixed yet, I had tried this locally but you're right it's not required/correct.
I'm working on fixing it, I'll revert the commit for now.